fireapp/lib/presentation/volunteer_information/volunteer_information.dart

```diff
@@ -0,0 +1,184 @@
+import 'dart:convert';
+
+import 'package:fireapp/base/widget.dart';
+import 'package:fireapp/domain/models/volunteer_information.dart';
+import 'package:fireapp/layout/wrapper.dart';
+import 'package:fireapp/presentation/fireapp_page.dart';
+import 'package:fireapp/presentation/volunteer_information/volunteer_information_viewmodel.dart';
+import 'package:fireapp/widgets/request_state_widget.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:fireapp/presentation/volunteer_information/volunteer_information_widget.dart';
+
+class VolunteerInformationPage extends StatefulWidget {
+  final String volunteerId;
+
+  const VolunteerInformationPage({
+    super.key,
+    required this.volunteerId,
+  });
+
+  @override
+  State createState() => _VolunteerInformationState();
+}
+
+class _VolunteerInformationState extends FireAppState<VolunteerInformationPage>{
+  String formatHours(List<List<int>> hours) {
+    return hours.map((hourRange) => '${formatHour(hourRange[0])} to ${formatHour(hourRange[1])}').join(', ');
+  }
+
+  String formatHour(int hour) {
+    if (hour == 0) {
+      return '12am';
+    } else if (hour < 12) {
+      return '$hour' + 'am';
+    } else if (hour == 12) {
+      return '12pm';
+    } else {
+      return '${hour - 12}' + 'pm';
+    }
+  }
+
+  @override
+  VolunteerInformationViewModel viewModel = GetIt.instance.get();
+
+  @override
+  void initState() {
+    super.initState();
+    viewModel.getVolunteerInformation(widget.volunteerId);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(AppLocalizations.of(context)?.volunteerInformationTitle ??
+            'Volunteer Information'),
+      ),
+        body: RequestStateWidget.stream<VolunteerInformation>(
+            state: viewModel.volunteerInformation,
+            retry: () => viewModel.getVolunteerInformation(widget.volunteerId),
+            child: (_, volunteerInformation) {
+              return Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    volunteer_information_widget(
```

Comment:
As per the design, each discrete group of information should be inside a container holding the surface color (from the theme) and a slight padding. Please do this (so ID would be it's own thing, same with first and last name, but availability would all be in the same group).